docs/en/CLITools.md

```diff
@@ -0,0 +1,1217 @@
+# Instructions on the use of mqadmin Management tools
+
+Before introducing the mqadmin management tool, first declare the following points:
```

Comment:
"the following points need to be declared. " seems better since there is no explicit Subject)